warn-stranded.lua

```diff
@@ -334,7 +334,8 @@ local function getStrandedUnits()
                 or 0
         end
 
-        if unitIgnored(unit) then
+        -- Ignore units who are gathering plants to avoid errors with stepladders
+        if unitIgnored(unit) or (unit.job.current_job and unit.job.current_job.job_type == df.job_type.GatherPlants) then
```

Comment:
I suggest also ignoring mining job types since they can un-strand themselves. There's a handy attr in df.job_types you can check instead of having to itemize all the dig job types

Reply:
`df.job_type.attrs[unit.job.current_job.job_type].type == 'Digging'`

Reply:
Gotcha, I'll get that added tonight